### PR TITLE
Login -> Login to Publish

### PR DESCRIPTION
**Pull request by jayenashar**
so people don't think they need to login to download

### CODE REVIEW

**Comment by timlinux** [comment]
Dear @jayenashar 

Thank you so much for your patch! Can I suggest that we rather implement this as a tooltip on the Login menu Item, Screen real estate on the menu bar is precious and I would not like to use such a long piece of text for one menu item.

Thanks!

Regards

Tim

**Comment by jayenashar** [comment]
yeah i was worried about that myself, too.  i'm not sure people will see a tooltip, but i also haven't seen that many people request an account for downloading.  (i was told there are many such requests.)

i was also thinking to change the button to simply "Publish" but someone said there are other reasons to login and it wouldn't be clear for everyone that you can login with the publish button or the top-right menu.

i'm ok to close this, as i'm working on changing the account request page to address this

**Comment by jayenashar** [comment]
to add some more context, this is the kind of email we received today:

> I would like to request a mantra so that I can create a user ID. 

this was from a person who wanted to upload a plugin, but both other emails we received in the last 24 hours were from people wanting to download plugins.

any help you can provide on the site to get us help to registering only necessary users would be appreciated.

**Comment by timlinux** [comment]
@jayenashar OK thanks for your feedback. Maybe we can add some text on the landing page indicating that you do not need an account to download resources, only to upload?

**Comment by jayenashar** [comment]
not sure which page you mean.  https://id.osgeo.org/ldap/create says:

> Note that you do NOT need an ID to download and use OSGeo software! 
> ...
> And BTW you don't need an account to download qgis plugins.

**Comment by jayenashar** [comment]
@timlinux i now understand the issue and why people think they need an osgeo account.  the login button is next to the search field in the header and there is no search button.  so people enter some text into the search field and then click the button next to it.  how should we fix this?

cc @strk

**Comment by strk** [comment]
cc @rduivenvoorde (we must have found the problem)

File: qgis-app/templates/base.html [hunk 1]
```diff
@@ -64,7 +64,7 @@
                             {% if user.is_authenticated %}
                             <li><a href="{% url "logout" %}">{% trans "Logout" %}</a></li>
                             {% else %}
-                            <li><a href="{% url "login" %}">{% trans "Login" %}</a></li>
+                            <li><a href="{% url "login" %}">{% trans "Login to Publish" %}</a></li>
```
Comment (strk) [inline qgis-app/templates/base.html:67]: How about just "Publish" ?

Comment (strk) [reply qgis-app/templates/base.html:67]: Assuming they will read it. Would a "Search" button *before* the "Publish" one help ?

Comment (jayenashar) [reply qgis-app/templates/base.html:67]: > Assuming they will read it. Would a "Search" button _before_ the "Publish" one help ?

i was thinking after the input field and before the login button

Screen Shot 2021-04-13 at 09 14 06

Comment (jayenashar) [reply qgis-app/templates/base.html:67]: > How about just "Publish" ?

I thought there may be other reasons to login so i didn't want to remove the word "login"

Comment (Gustry) [reply qgis-app/templates/base.html:67]: Yes, for reviewing, editing existing plugin, mark as deprecated a plugin etc.
So I think we should keep "login".

But maybe change this PR to only add a "search button".

@jayenashar Good catch for the UX issue !

Comment (jayenashar) [reply qgis-app/templates/base.html:67]: will close this PR and make a new one with a search button (when i have a chance)

**Pull request closed without merging**